.travis.yml

```diff
@@ -0,0 +1,32 @@
+
+language: node_js
+
+node_js:
+ - "stable"
+
+services:
+ - postgresql
+
+before_install:
+ - npm install sequelize-cli -g
+
+env:
+ global:
+   - TEST_DATABASE_URL=postgres://postgres:@127.0.0.1:5432/test_database
+   - SECRET_KEY=secretkey
+   - NODE_ENV=test
+   - SOCKET_PORT=3000
```

Comment:
@davidfrank96 We should not be displaying our global variables like this. Travis provides a key/value pair store where these variables can be stored and then referenced here. Please look into that.

Reply:
thank you for the kind words.. will implement that ASAP